Scripts/DownloadCargoBinaryFromGitHub/DownloadCargoBinaryFromGitHub.py

```diff
@@ -44,8 +44,8 @@
 # Download assets
 for asset in releases[0]['assets']:
     name = asset['name'].lower()
-    if ("x86_64-pc-windows-msvc" in name or "x86_64-unknown-linux-gnu" in name
-       and asset['name'].endswith(('.zip', '.tar.gz'))):
+    if (("x86_64-pc-windows-msvc.full" in name or "x86_64-unknown-linux-gnu.full" in name)

```

Comment:
Is `".full"` expected to work with all releases?

Reply:
Updated - I pushed the commit that removed this to the wrong remote. locally, it works fine without the ".full". Running it in pipeline format with ".full" removed, just to be sure.

Reply:
Pipeline was successful.